### PR TITLE
Corrected script for some item #12

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -68422,6 +68485,30 @@ Body:
...
+      bonus bHealPower,15;
+      bonus bAddItemHealRate,15;
+      if (.@r>=7) {
+         bonus bBaseAtk,15;
+         bonus bMatk,15;
+      }
+      if (.@r>=7) {
+         bonus bBaseAtk,20;
+         bonus bMatk,20;
+      }
```
Comment (Atemo) [inline db/re/item_db_equip.yml:68511]:
```suggestion
         if (.@r>=9) {
            bonus bBaseAtk,20;
            bonus bMatk,20;
         }
      }
```

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -29392,3 +29392,978 @@ Body:
...
+    Script: |
+      bonus2 bAddRace2,RC2_BioLab,20;
+      bonus2 bMagicAddRace2,RC2_BioLab,20;
+  - Combos:
+      - Combo:
+          - Record_Sward_TW   # 490094
+          - Subject_Cape_TW   # 480024
+    Script: |
+      if (getequiprefinerycnt(EQI_GARMENT) >= 10) {
+         if (eaclass()&EAJL_THIRD && BaseJob == Job_Swordman) {
```
Comment (Atemo) [inline db/re/item_combos.yml:30023]:
```suggestion
         if (eaclass()&EAJL_THIRD && BaseJob == Job_Knight) {
```

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -29392,3 +29392,978 @@ Body:
...
+    Script: |
+      bonus2 bAddItemHealRate,569,100;
+      bonus2 bAddMonsterDropItem,569,50;
+  - Combos:
+      - Combo:
+          - Santa's_Shoes  # 22068
+          - Mole_Card    # 4343
+    Script: |
+      if (getequiprefinerycnt(EQI_SHOES) >= 6)
+         bonus2 bSkillUseSP,"MC_CARTREVOLUTION",4;
```
Comment (Atemo) [inline db/re/item_combos.yml:30369]: Missing newline

**Review by Atemo — COMMENTED** [review]
Good work!

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_combos.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

**Pull request merged**